File: Daily Flash/week 4/day 5/MySolutions/Python/Program5.py
print("\n ------- Pattern ------- \n\n")

for i in range(4):
	val=1
	for j in range(4):
		if( i==j ):
			print("   =",end=" ")
			val=val+1
		else:
			print("  ",val,end=" ")
			val=val+1
		
	print("\n")	
	
	
'''

Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/Python$ python3 Program5.py

 ------- Pattern ------- 


   =    2    3    4 

   1    =    3    4 

   1    2    =    4 

   1    2    3    = 


'''	
	
		

File: Daily Flash/week 4/day 5/MySolutions/C/Program5.c
#include<stdio.h>

void main() {

	printf("\n  --------  Pattern -------   \n\n");
	int val=1;
	
	for(int i=0;i<4;i++) {
	
		val=1;
		for(int j=0;j<4;j++) {
		
			if( i==j ) {
				printf("   =");
				val++;
			}
			else	
				printf("   %d",val++);
		}
		
		printf("\n\n");
	}
	
}


/*
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/C$ cc Program5.c
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 5/MySolutions/C$ ./a.out

  --------  Pattern -------   

   =   2   3   4

   1   =   3   4

   1   2   =   4

   1   2   3   =


*/
